engine/src/test/java/io/camunda/zeebe/engine/processing/bpmn/activity/NativeUserTaskTest.java

```diff
@@ -608,6 +609,42 @@ public void shouldClaimUserTask() {
         .hasAssignee("foo");
   }
 
+  @Test
+  public void shouldUpdateUserTaskAttributes() {
+    // given
+    ENGINE
+        .deployment()
+        .withXmlResource(
+            process(
+                t ->
+                    t.zeebeCandidateGroups("foo, bar")
+                        .zeebeCandidateUsers("oof, rab")
+                        .zeebeFollowUpDate("2023-03-02T15:35+02:00")
+                        .zeebeDueDate("2023-03-02T16:35+02:00")))
+        .deploy();
+    final long processInstanceKey = ENGINE.processInstance().ofBpmnProcessId(PROCESS_ID).create();
+
+    // when
+    ENGINE.userTask().ofInstance(processInstanceKey).update(new UserTaskRecord());
```

Comment:
:thinking: I wonder if we should assert the initial values of the `UserTaskRecord`. This would allow us to detect any regressions when some of these properties are not set.

Reply:
I thought about it as well and wasn't sure if that's overkill. Since we both thought about it now, I have enough evidence to actually add it. 😄